`ErrNoSuchField` is undefined

This does not build.

```
./adifrecord.go:118:16: undefined: ErrNoSuchField
```

_Originally posted by @xylo04 in https://github.com/Matir/adifparser/issues/14#issuecomment-1295118845_

--- a/adifrecord.go
+++ b/adifrecord.go
@@ -115,5 +115,5 @@ func (r *baseADIFRecord) DeleteField(name string) (bool, error) {
 		delete(r.values, name)
 		return true, nil
 	}
-	return false, ErrNoSuchField
+	return false, NoSuchField
 }

